packages/pipeline-services/src/migration/migrateV1/index.ts

```diff
@@ -15,10 +15,7 @@
  */
 
 const hasAppdataField = (node: any, fieldName: string): boolean => {
-  return (
-    Object.prototype.hasOwnProperty.call(node, "app_data") &&
-    Object.prototype.hasOwnProperty.call(node["app_data"], fieldName)
-  );
+  return !!node.app_data && !!node.app_data.hasOwnProperty(fieldName);
```

Comment:
why not use nullish coalescing?

Reply:
like with the `?` operator:
```ts
!!node.app_data?.hasOwnProperty(fieldName)
```

Reply:
because I check `!!node.app_data` right before, so the boolean would short-circuit before it get there. I can add it, but I didn't see a reason to

Reply:
ah nvm, I misunderstood you, yes that would simplify the line, I'll go update it